library/src/main/java/com/owncloud/android/lib/resources/status/GetCapabilitiesRemoteOperation.java

```diff
@@ -425,6 +427,10 @@ private OCCapability parseResponse(String response) throws JSONException {
                                 respFiles.getBoolean(PROPERTY_VERSIONING)));
                     }
 
+                    if (respFiles.has(PROPERTY_LOCKING)) {
```

Comment:
Please add a test in GetCapabilitiesIT.
Like in line 134 you can bind this to NC24.